contrib/ohi-release-notes/action.yml

```diff
@@ -40,41 +49,58 @@ runs:
         git-root: ${{ inputs.git-root }}
         markdown: ${{ inputs.git-root }}/CHANGELOG.md
         yaml: ${{ inputs.git-root }}/changelog.yaml
+    - name: Check if the release is empty
+      id: empty
+      uses: newrelic/release-toolkit/is-empty@v1
+      with:
+        yaml: ${{ inputs.git-root }}/changelog.yaml
+    - name: Abort releasing if the changelog is empty
+      if: ${{ steps.empty.outputs.is-empty == 'true' && inputs.fail-if-empty == 'true' }}
+      shell: bash
+      run: |
+        echo "Aborting release, changelog is empty is-empty=${{ steps.empty.outputs.is-empty }}"
+        exit 0
     - name: Check if the release is held
+      if: ${{ steps.empty.outputs.is-empty != 'true' }}
       id: held
       uses: newrelic/release-toolkit/is-held@v1
       with:
         yaml: ${{ inputs.git-root }}/changelog.yaml
     - name: Abort releasing if the release is held
-      if: ${{ steps.held.outputs.is-held == 'true' && inputs.fail-if-held == 'true' }}
+      if: ${{ steps.held.outputs.is-held == 'true' && inputs.fail-if-held == 'true' && steps.empty.outputs.is-empty != 'true' }}
```

Comment:
> why adding steps.empty.outputs.is-empty != 'true'?

As I understand it we don't want to keep running following steps either if `is-empty` or `is-held` is true. Then,  since `is-empty` comes first (completely arbitrarily chosen), in the case it already failed we no longer need to check the `is-held`.

Reply:
I decoupled the behaviour of `is-empty` and `is-held` in this commit c39b834e4d8104fe9cc65195f168425e57935d3e. Now, they are evaluated as 2 independent actions.